loading-indicator.ios.ts

```diff
@@ -0,0 +1,50 @@
+import {Color} from 'color';
+import {OptionsI} from './loading-indicator.common';
+
+declare var MBProgressHUD: any;
+declare var MBProgressHUDModeCustomView: any;
+declare var UIApplication: any;
+declare var UIImageView: any;
+declare var UIImage: any;
```

Comment:
Are these needed such that we don't get warnings from typescript not knowing about those constants?